frappe/public/js/frappe/file_uploader/FileUploader.vue

```diff
@@ -488,8 +488,8 @@ function upload_files(dialog) {
 		return Promise.reject();
 	}
 
-	dialog?.get_primary_btn().prop("disabled", true);
-	dialog?.get_secondary_btn().prop("disabled", true);
+	dialog?.get_primary_btn?.().prop("disabled", true);
+	dialog?.get_secondary_btn?.().prop("disabled", true);
```

Comment:
Why are these not defined? :thinking: 

https://github.com/frappe/frappe/blob/501297b71c5eca289ac55457561fa972b5b1ccf7/frappe/public/js/frappe/ui/dialog.js#L155-L157

Reply:
It's because `dialog` is the first parameter of the function, which is also used as an event listener: when Vue attaches the event handler (`upload_files`), then you have `dialog = MouseEvent` because of the implicit first parameter `$event`.

https://github.com/frappe/frappe/blob/0934dfea93c22e111bd0c2b836273bc5b78adc9e/frappe/public/js/frappe/file_uploader/FileUploader.vue#L191

```vue
becomes
<button class="btn btn-primary btn-sm margin-right" @click="$event => upload_files($event)">
```

https://github.com/frappe/frappe/pull/25034#issuecomment-1961405668